db/migrate/20161229080315_add_comments_count_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddCommentsCountToEvents < ActiveRecord::Migration
+  def change
+    add_column :events, :comments_count, :integer, default: 0, null: false
```

Comment:
So this PR looks good. The only thing we'd be missing which I just noticed is that we need to populate the `comments_count` field of existing events.